modules/cluster-logging-manual-rollout-rolling.adoc

```diff
@@ -15,6 +15,8 @@ runs requires a reboot.
 
 * Cluster logging and Elasticsearch must be installed.
 
+* Install the {product-title} link:https://github.com/openshift/origin-aggregated-logging/tree/master/elasticsearch#es_util[*es_util*] tool
```

Comment:
How is this 'installed'? It looks like its part of the image we ship / support. So is this really a pre-resquisite?